Documentation/common-problems.md

```diff
@@ -4,22 +4,66 @@ weight: 78
+1. Kubernetes status and logs
+1. Ceph status
+
+## Kubernetes Tools
+Kubernetes status will be the first line of investigating when something goes wrong with the cluster. Here are a few artifacts that would be helpful to gather:
+- Rook pod status: 
+  - `kubectl get pod -n rook -o wide`
+  - `kubectl get pod -n rook-system -o wide`
+- Logs for Rook pods
+  - Logs for the operator: `kubectl logs -n rook-system -l app=rook-operator`
+  - Logs for a specific pod: `kubectl logs -n rook <pod-name>`, or a pod using a label such as mon1: `kubectl logs -n rook -l mon=rook-ceph-mon1`
+  - Logs on a specific node to find why a PVC is failing to mount:
+    - Rook agent errors around the attach/detach: `kubectl logs -n rook-system <rook-agent-pod>`
+    - Connect to the node, then get kubelet logs (if your distro is using systemd): `journalctl -u kubelet`
+- Other Rook artifacts:
+  - The monitors that are expected to be in quorum: `kubectl -n rook get configmap rook-ceph-mon-endpoints -o yaml | grep data`
+  - More artifacts in the `rook` namespace: `kubectl -n rook get all`
+  
+## Ceph Tools
+After you verify the basic health of the running pods, next you will want to run Ceph tools for status of the storage components. There are two ways to run the Ceph tools, either in the Rook toolbox or inside other Rook pods that are already running.
+
+### Tools in the Rook Toolbox
+ The [rook-tools pod](./toolbox.md) is a one-stop shop for both Ceph tools and other troubleshooting tools. Once the pod is up and running one can `kubectl exec` into the pod to execute Ceph commands to evaluate that current state of the cluster. 
+
+### Tools in other pods
+The Ceph tools are found in all of the Rook pods where Ceph is running, such as the operator, monitors, and OSDs. Rather than starting the toolbox pod, you can connect to the existing pods to more quickly execute the Ceph tools. For example, to connect to the operator pod:
+
+```bash
+# Find the name of the rook operator pod
+kubectl -n rook-system get pod -l app=rook-operator
+
+# Connect to the operator pod
+kubectl -n rook-system exec -it <pod> bash
+
+# For example:
+# kubectl -n rook-system exec -it rook-operator-77cf655476-vfspf bash
+```
+
+Now from inside the operator pod you can execute the Ceph tools.
+
+It is preferred to connect to the operator pod rather than monitors and OSDs since the operator always has the latest configuration files. The mon and OSD pods will only re-create the config files when the pod is restarted. If a monitor fails over, the config in the other mon and osd pods may be out of date. This could cause your ceph commands to timeout connecting to invalid mons. Note that the running mon and osd daemons are not affected by this issue because of their live connection to the mon quorum. If you have multiple Rook clusters, it is preferred to connect to the Rook toolbox for a specific cluster. Otherwise, your ceph commands may connect to the wrong cluster.
```

Comment:
i think the details in this wall of text aren't really important.  i would trim it down to just:

> It is preferred to connect to the **operator** pod rather than monitors and OSDs since the operator always has the latest configuration files.  If you have multiple Rook clusters, it is preferred to connect to the Rook **toolbox** for a specific cluster. Otherwise, your ceph commands may connect to the wrong cluster.

Reply:
True, that's more of a code level comment, thanks